deliver/lib/deliver/runner.rb

```diff
@@ -216,15 +216,15 @@ def upload_binary
           package_path: "/tmp",
           platform: platform
         )
-        result = transporter.upload(package_path: package_path, asset_path: ipa_path)
+        result = transporter.upload(package_path: package_path, asset_path: ipa_path, platform: platform)
       when "osx"
         package_path = FastlaneCore::PkgUploadPackageBuilder.new.generate(
           app_id: Deliver.cache[:app].id,
           pkg_path: pkg_path,
           package_path: "/tmp",
           platform: platform
         )
-        result = transporter.upload(package_path: package_path, asset_path: pkg_path)
+        result = transporter.upload(package_path: package_path, asset_path: pkg_pat, platform: "macos")
```

Comment:
since altool needs a option `-t` and it accepts `ios`, `macos` or `appletvos`. `osx` cannot be passed so I convert it here. but it may be better to do it inside build command construction function. let me fix it